Sources/Client/Client.cpp

```diff
@@ -55,7 +55,7 @@
 
 #include "NetClient.h"
 
-DEFINE_SPADES_SETTING(cg_chatBeep, "1");
+DEFINE_SPADES_SETTING(cg_chatBeep, "100");
```

Comment:
The existing users will keep using the old value of `cg_chatBeep`, so changing the value range or the default value is not recommended. I suggest using a 0–1 range (with intermediate values).